fdbserver/DataDistribution.actor.cpp

```diff
@@ -162,6 +163,9 @@ class TCMachineInfo : public ReferenceCounted<TCMachineInfo> {
 
 ACTOR Future<Void> updateServerMetrics(Reference<TCServerInfo> server);
 
+// Read storage metadata from database, and do necessary updates
```

Comment:
reading the comment it gives an impression that routine does more than just "checks"; if updates are performed shouldn't the routine name be "readAndUpdateStorateMetadata" ?